config/toml.go

```diff
@@ -583,6 +583,12 @@ initial_block_retain_height = {{ .Storage.Pruning.DataCompanion.InitialBlockReta
 # already set a block results retain height, this is ignored.
 initial_block_results_retain_height = {{ .Storage.Pruning.DataCompanion.InitialBlockResultsRetainHeight }}
 
+# If set to true, CometBFT will force compaction to happen for databases that support this feature
+compact_on_pruning = true
```

Comment:
From the PR description it says it's `false` by default. Just want to confirm if `true` should be here. If true, then my comment above needs to be updated to say default is `false`

Reply:
Should be false, will edit.